universe/world/calendar/calendar.go

```diff
@@ -0,0 +1,195 @@
+package calendar
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/google/uuid"
+	"github.com/momentum-xyz/posbus-protocol/posbus"
+	"github.com/pkg/errors"
+
+	"github.com/momentum-xyz/ubercontroller/logger"
+	"github.com/momentum-xyz/ubercontroller/types/entry"
+	"github.com/momentum-xyz/ubercontroller/types/generic"
+	"github.com/momentum-xyz/ubercontroller/universe"
+	"github.com/momentum-xyz/ubercontroller/utils"
+)
+
+type Calendar struct {
+	ctx      context.Context
+	world    universe.World
+	timerSet *generic.TimerSet[string]
+}
+
+type Event struct {
+	SpaceID *uuid.UUID
+	Title   string
+	Start   time.Time
+	End     time.Time
+	EventID string
+}
+
+var pluginID = uuid.MustParse("f0f0f0f0-0f0f-4ff0-af0f-f0f0f0f0f0f0")
+var attributeName = "events"
```

Comment:
your feature request was done, use it
https://github.com/momentum-xyz/ubercontroller/blob/develop/universe/types.go#L33